turbobt/substrate/pallets/author.py

```diff
@@ -10,14 +10,20 @@
 from .state import RuntimeVersion
 
 
+class Era(typing.TypedDict):
+    period: int
+    phase: int | None
+    current: int | None
+
+
 class Author(Pallet):
     async def submitAndWatchExtrinsic(
         self,
         call_module: str,
         call_function: str,
         call_args: dict[str, typing.Any],
         key: bittensor_wallet.Keypair,
-        era: dict | None = None,
+        era: Era | None = ...,
```

Comment:
I think the type hint is wrong here, cause `...` here is neither an instance of `None` nor is it and `Era`